ostd/src/mm/page_table/node/entry.rs

```diff
@@ -38,11 +39,11 @@ impl<'a, E: PageTableEntryTrait, C: PagingConstsTrait> Entry<'a, E, C> {
         self.pte.is_present() && !self.pte.is_last(self.node.level())
     }
 
-    /// Gets a owned handle to the child.
-    pub(in crate::mm) fn to_owned(&self) -> Child<E, C> {
+    /// Gets a reference to the child.
+    pub(in crate::mm) fn to_ref(&self) -> Child<'_, E, C> {
```

Comment:
Following the above comment, this should be able to return `Child<'rcu, E, C>`.

Then please drop the `PageTableGuard::from_raw_paddr` and `PageTableGuard::into_raw_paddr`. We should just introduce a `PageTableRef::make_guard_unchecked` method to convert a `PageTableRef` to a `PageTableGuard`. The compiler should check the lifetime.